native/src/components/MapView.tsx

```diff
@@ -15,11 +15,28 @@ const StyledMap = styled(MapboxGL.MapView)`
 
 type MapViewPropsType = {
   boundingBox: BBox
+  featureCollection: FeatureCollection
+}
+
+const textOffsetY = 1.25
+const layerProps: SymbolLayerProps = {
+  id: 'point',
+  style: {
+    symbolPlacement: 'point',
+    iconAllowOverlap: true,
+    iconIgnorePlacement: true,
+    iconImage: ['get', 'symbol'],
+    textField: ['get', 'title'],
+    textFont: ['Roboto Regular'],
+    textOffset: [0, textOffsetY],
+    textAnchor: 'top',
+    textSize: 12
+  }
 }
```

Comment:
Just for my understanding: Is this something that will be replaced at a later point by the style spec served from our servers in this issue? Or will this stay? https://issues.tuerantuer.org/browse/IGAPP-535

Reply:
Could be, this is open for discussion I'd say. By moving this into the style.json we could reduce code duplication if we show the map in native and web.

Reply:
I'd say it is better to share what is the same. I'am not sure about the fonts though. As we use different fonts in native and web right?